tests/operators/test_python.py

```diff
@@ -847,31 +847,41 @@ def test_virtualenv_serializable_context_fields(self, create_task_instance):
         assert set(context) == declared_keys
 
     @pytest.mark.parametrize(
-        "extra_kwargs, actual_exit_code, expected_state",
+        "kwargs, actual_exit_code, expected_state",
         [
-            (None, 99, TaskInstanceState.FAILED),
+            ({}, 0, TaskInstanceState.SUCCESS),
+            ({}, 100, TaskInstanceState.FAILED),
+            ({}, 101, TaskInstanceState.FAILED),
+            ({"skip_on_exit_code": None}, 0, TaskInstanceState.SUCCESS),
+            ({"skip_on_exit_code": None}, 100, TaskInstanceState.FAILED),
+            ({"skip_on_exit_code": None}, 101, TaskInstanceState.FAILED),
+            ({"skip_on_exit_code": 100}, 0, TaskInstanceState.SUCCESS),
             ({"skip_on_exit_code": 100}, 100, TaskInstanceState.SKIPPED),
-            ({"skip_on_exit_code": [100]}, 100, TaskInstanceState.SKIPPED),
-            ({"skip_on_exit_code": (100, 101)}, 100, TaskInstanceState.SKIPPED),
             ({"skip_on_exit_code": 100}, 101, TaskInstanceState.FAILED),
+            ({"skip_on_exit_code": 0}, 0, TaskInstanceState.SKIPPED),
```

Comment:
Look like this test is failing in main

Reply:
#36367

Reply:
Hmm, why CI didn't fail in this PR, and failed only after merge?

Reply:
That's a good question. Let me see.

Reply:
Ah. Because our "selective checks" are too selective. The changes were done in "python" operator but the failing tests are now separated into `BranchExternalPython`, `BranchPythonVenv`, `PythonVenv`, `ExternalPython` -> this allows better parallelism. We are missing implicit dependency to trigger all the "sub-types" when just `Python`changes. Let me add a dependency real quick for the future.

Reply:
Fix here: https://github.com/apache/airflow/pull/36372